inc/views/base/parser/video.twig

```diff
@@ -0,0 +1,21 @@
+{% if type == 'dailymotion' %}
```

Comment:
I wonder if it's worth splitting these out into partial templates? This seems a bit messy to me personally.

Reply:
Having 10+ templates for such a repetitive thing doesn't look clean personally, but I can push a PR if you want :)

Reply:
We can leave it as is for now, and if people provide feedback against it, we can always split them out easily at a later date - with twig we don't need PHP changes anymore afterall 😉